milestone-3/frontend/src/recipeData.ts

```diff
@@ -0,0 +1,91 @@
+export interface Recipe {
+  name: string;
+  description: string;
+  image: string;
+  ingredients: string[];
+  instructions: string[];
+}
```

Comment:
you can move this to a types.ts file or something similar (or keep it here) and import this interface wherever you need it